mobile/src/main/java/org/openhab/habdroid/util/Constants.java

```diff
@@ -10,27 +10,30 @@
 package org.openhab.habdroid.util;
 
 public class Constants {
-    public static final String PREFERENCE_SCREENTIMEROFF      = "default_openhab_screentimeroff";
-    public static final String PREFERENCE_LOCAL_USERNAME      = "default_openhab_username";
-    public static final String PREFERENCE_LOCAL_PASSWORD      = "default_openhab_password";
-    public static final String PREFERENCE_REMOTE_PASSWORD     = "default_openhab_remote_password";
-    public static final String PREFERENCE_REMOTE_USERNAME     = "default_openhab_remote_username";
-    public static final String PREFERENCE_SITEMAP             = "default_openhab_sitemap";
-    public static final String PREFERENCE_SSLCERT             = "default_openhab_sslcert";
-    public static final String PREFERENCE_SSLHOST             = "default_openhab_sslhost";
-    public static final String PREFERENCE_ALTURL              = "default_openhab_alturl";
-    public static final String PREFERENCE_URL                 = "default_openhab_url";
-    public static final String PREFERENCE_THEME               = "default_openhab_theme";
-    public static final String PREFERENCE_ANIMATION           = "default_openhab_animation";
-    public static final String PREFERENCE_DEMOMODE            = "default_openhab_demomode";
-    public static final String PREFERENCE_FULLSCREEN          = "default_openhab_fullscreen";
-    public static final String PREFERENCE_TONE                = "default_openhab_alertringtone";
-    public static final String PREFERENCE_CLEAR_CACHE         = "default_openhab_cleacache";
-    public static final String PREFERENCE_SSLCLIENTCERT       = "default_openhab_sslclientcert";
-    public static final String PREFERENCE_SSLCLIENTCERT_HOWTO = "default_openhab_sslclientcert_howto";
-    public static final String PREFERENCE_DEBUG_MESSAGES      = "default_openhab_debug_messages";
-    public static final String DEFAULT_GCM_SENDER_ID          = "737820980945";
-    public static final String PREFERENCE_COMPAREABLEVERSION  = "versionAsInt";
+    public static final String PREFERENCE_SCREENTIMEROFF        = "default_openhab_screentimeroff";
```

Comment:
A general question into the round: Do we actually need these whitespaces to format this like a table (so that all values start at the same column)? It makes the git diffs unreadable in my opinion :P

Reply:
I think we dont need it. You have to increase it every time you introduce a long new variable.